From f02d92ba536cf3b9aa0dae45667fb35a2a5e93cd Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 26 Nov 2019 15:58:50 +0800
Subject: [PATCH] ARM: dts: rockchip: rk312x: Add SYS_STATUS_REBOOT for dmc

Add SYS_STATUS_REBOOT for dmc so that the frequency and voltage will
change when reboot.

Change-Id: If26c704bfda361d03cf3a8b0e16781902ddf7e12
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm/boot/dts/rk312x.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk312x.dtsi b/arch/arm/boot/dts/rk312x.dtsi
index 50fa2fa5135e..a1a2c0f90da4 100644
--- a/arch/arm/boot/dts/rk312x.dtsi
+++ b/arch/arm/boot/dts/rk312x.dtsi
@@ -230,6 +230,7 @@
 			/*system status		freq(KHz)*/
 			SYS_STATUS_NORMAL	456000
 			SYS_STATUS_SUSPEND	200000
+			SYS_STATUS_REBOOT	396000
 		>;
 		auto-min-freq = <300000>;
 		auto-freq-en = <0>;
-- 
2.35.3

